src/main/java/jenkins/plugins/office365connector/Webhook.java

```diff
@@ -62,6 +64,7 @@ public DescriptorImpl getDescriptor() {
     @DataBoundConstructor
     public Webhook(String url) {
         this.url = StringUtils.isEmpty(url) ? getDescriptor().getGlobalUrl() : url;
+        this.setProxyPluginConfiguration(getDescriptor().getIp(), getDescriptor().getPort(), getDescriptor().getUsername(), getDescriptor().getPassword());
```

Comment:
getDescriptor() can be invoked once and result assigned to temp reference

Reply:
I've resolved this as follows.

```
    @DataBoundConstructor
    public Webhook(String url) {
        this.url = StringUtils.isEmpty(url) ? getDescriptor().getGlobalUrl() : url;

        DescriptorImpl globalConfig = getDescriptor();
        this.setProxyPluginConfiguration(globalConfig.getIp(), globalConfig.getPort(), globalConfig.getUsername(), globalConfig.getPassword());
    }
```